### PR TITLE
New String Variable to avoid translation problem

### PR DESCRIPTION
**Pull request by luveluen**
"optOut" is already used to skip class selection
recently the variable was also used for the checkbox to disable ingame messages.

A phrase equal to "Opt Out" does not exist in all languages, we need two different strings for 
"No, I don't want to choose a class yet" and "Disable messages".
I created optOutMsg for this purpose. (they have the same value in English).

### CODE REVIEW

File: common/locales/en/character.json [hunk 1]
```diff
@@ -126,6 +126,7 @@
     "rogueText": "Rogues love to accumulate wealth, gaining more Gold than anyone else, and are adept at finding random items. Their iconic Stealth ability lets them duck the consequences of missed Dailies. Play a Rogue if you find strong motivation from Rewards and Achievements, striving for loot and badges!",
     "healerText": "Healers stand impervious against harm, and extend that protection to others. Missed Dailies and bad Habits don't faze them much, and they have ways to recover Health from failure. Play a Healer if you enjoy assisting others in your Party, or if the idea of cheating Death through hard work inspires you!",
     "optOut": "Opt Out",
+    "optOutMsg: "Opt Out",
```
Comment (paglias) [inline common/locales/en/character.json]: @luveluen there's a missing " before the semicolon `"optOutMsg: "Opt Out"` should be `"optOutMsg": "Opt Out"`

**Comment by Alys** [comment]
@luveluen Would it be obvious to the translators which string was used for each purpose?

I'm not sure `optOutMsg` clearly indicates it's related to the private message system because you could interpret "Msg" as meaning this is a generic message about opting out of things. Use something like `optOutOfPMs` maybe? We might also want to change `optOut` to `optOutOfClassSystem` or similar. That would help avoid this same problem in future for other things you can opt out of.

**Comment by luveluen** [comment]
"optOutOfClasses" and "optOutOfPMs" are now the key names.
The quotation mark mistake is fixed.

File: common/locales/en/character.json [hunk 2]
```diff
@@ -125,7 +125,8 @@
...
     "rogueText": "Rogues love to accumulate wealth, gaining more Gold than anyone else, and are adept at finding random items. Their iconic Stealth ability lets them duck the consequences of missed Dailies. Play a Rogue if you find strong motivation from Rewards and Achievements, striving for loot and badges!",
     "healerText": "Healers stand impervious against harm, and extend that protection to others. Missed Dailies and bad Habits don't faze them much, and they have ways to recover Health from failure. Play a Healer if you enjoy assisting others in your Party, or if the idea of cheating Death through hard work inspires you!",
-    "optOut": "Opt Out",
+    "optOutOfClasses": "Opt Out",
+    "optOutOfPMs": "Opt Out",
     "optOutText": "Can't be bothered with classes? Want to choose later? Opt out - you'll be a warrior with no special abilities. You can read about the class system later on the wiki and enable classes at any time under User -> Stats.",
```
Comment (crookedneighbor) [inline common/locales/en/character.json:130]: This should be changed to `optOutOfClassesText`

**Comment by crookedneighbor** [comment]
I've corrected the `optOutText` string and merged this in.

This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by luveluen** [comment]
be0d25d1-9e71-400d-9d78-7c06647f419f
thank you

**Comment by crookedneighbor** [comment]
I've notated this toward your next tier.

**Pull request closed without merging**